### PR TITLE
feat(mail): Flag to toggle message logging itself

### PR DESCRIPTION
**Pull request by patrickvenetz**
Allows to disable logging a message in `mailLog` table. Disabled logging in mutation `submitInheritenceStory`.

It adds a templates name as a tag for more convenience in Mandrill dashboard.

- [ ] Remove existing message contents in `mailLog` table which was submitted via `submitInheritenceStory` mutation via SQL Stmt.

### CODE REVIEW

File: packages/crowdsourcing/graphql/resolvers/_mutations/submitInheritanceStory.js [hunk 1]
```diff
@@ -54,9 +54,15 @@ ${content ? 'Erbgeschichte: ' + content : ''}
...
-  }, context)
+  return sendMail(
+    {
+      to: CS_INHERITANCE_TO,
+      subject: CS_INHERITANCE_SUBJECT,
+      text: emailText
+    },
+    context,
+    {
+      logMessage: false
```
Comment (patte) [inline packages/crowdsourcing/graphql/resolvers/_mutations/submitInheritanceStory.js]:
```suggestion
      logMail: false
```
After thinking about it `logMail` feels better than `logMessage`. An email is an email and less a message. Or just shorten it to `log: false`

File: packages/mail/lib/mailLog.js [hunk 2]
```diff
@@ -54,32 +58,40 @@ const send = async ({
...
+    Object.assign(info, log.info)
+  }
+
+  if (template) {
+    info.template = template
+  }
+
+  if (logMessage) {
+    info.message = message
   }
```
Comment (patte) [inline packages/mail/lib/mailLog.js]: Why did you change this code block? What's the advantage of this notation to the old one?

Comment (patrickvenetz) [reply packages/mail/lib/mailLog.js]: Easier to test, read and stitch together.

File: packages/crowdsourcing/graphql/resolvers/_mutations/submitInheritanceStory.js (hunk 1 above)
Comment (patrickvenetz) [reply packages/crowdsourcing/graphql/resolvers/_mutations/submitInheritanceStory.js]: It still logs whole "email" just not it's content. And since content is wrapped in `message`, would like to stick to it.
